src/clib/initialize_rates.c

```diff
@@ -466,7 +528,105 @@ int initialize_rates(chemistry_data *my_chemistry, chemistry_data_storage *my_ra
+    my_rates->kz30 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz31 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz32 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz33 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz34 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz35 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz36 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz37 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz38 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz39 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz40 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz41 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz42 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz43 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz44 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz45 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz46 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz47 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz48 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz49 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz50 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz51 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz52 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz53 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->kz54 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+
+    my_rates->cieY06  = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+
+    my_rates->n_cr_n = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->n_cr_d1 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    my_rates->n_cr_d2 = malloc(my_chemistry->NumberOfTemperatureBins * sizeof(double));
+    
+    my_rates->k24 = 0;
+    my_rates->k25 = 0;
+    my_rates->k26 = 0;
+    my_rates->k27 = 0;
+    my_rates->k28 = 0;
+    my_rates->k29 = 0;
+    my_rates->k30 = 0;
+    my_rates->k31 = 0;
+    my_rates->piHI = 0;
+    my_rates->piHeII = 0;
+    my_rates->piHeI = 0;
+    my_rates->crsHI = 0;
+    my_rates->crsHeI = 0;
+    my_rates->crsHeII = 0;
+    my_rates->comp_xray = 0;
+    my_rates->temp_xray = 0;
+*/
```

Comment:
We can delete these, right?

```suggestion
/* GENCHIAKI ADDITIONAL RATES
*/
```

Reply:
Yes, we can delete them.